tests/id-test.js

```diff
@@ -3,8 +3,8 @@ var PeerId = require('../src')
-  pubKey: '-----BEGIN RSA PUBLIC KEY-----\nMIIBCgKCAQEAgPEiGHOwFEUdo95/DaALH69umbFI4xD3Jmla0hiHbkcW535arBfFd8nJs5VP\nt49sgdSgn1ZmiqmHLgMwMz6mKplu4GsmWj5mjdyxiNl5z6R2rF+ZziuiwRTeHVX/8zR8M7Cb\nh0QXmzpoq6LcNOFHbg495zsbmT9QAtjVsS1KyF5324mxbTZtjaD6hxJkAL8aVi0ikvhAL4Hu\nZ1m3brjSSZ0+epFCFL7UIoJlFfOvap2sAyxdOrSvY2PXKTE00s51YTin5+CrvofRLCJPROls\n1oFkhXIDGfuTGTxMxe3hSUlNj0LjQi6RXPGat/5XH0nCuFTODmyhrnnnx51OdgT9vwIDAQAB\n-----END RSA PUBLIC KEY-----\n'
+  privKey: 'CAASqQkwggSlAgEAAoIBAQC1/m95i4waplpSl43/uefQClPfuguPsX6qa6pWE0Df+krs5p+RnhX+8j2aWBI9QZ0pliGryet5qM3in6J1ihb2LzjLUWIsWrMLxtqi/mQDjj1f0HfV11Q/k2v6yVgSgYJ+VoW51hCpAEdEnhthsbyKWMD6xVIvtkqOmK/FaxUMa2boWCVKsycf39U6GVaiuTk32btw6LqGMo8P1cehWDy80SOzv0qk2mD7ZJbKnVfVm8xwU+gzogasOFYs/LmKb4IEV9Otq0MHmWGUN0VyC82htoGBidheRs1Ssh7TDVtWlNTJSG1vJJV7cnIhpGKut2b0pMxhY1Sg0kn/VrfriXINAgMBAAECggEBAJrJuI4r/hF8gz3T4NYri9oJrqSOW97vG8heohVrcrYM70TmMblsN1ELPxHS7lBjSgRgyGqP5lMnG1UwaMCHnlfseeWTZmhLDBVsH/CZZP8RL2oaqJGb/u/Dtwcp0FqNBCvn8vzH8IuMzRCzWJ6SyMTyD9A5m1kxNeBqRLUoClLwZsLKuL8ld9sXfwmn1D1yY9sWLWCcrlPZnm4v49otc6Q9GYpurhOLBlQOqkEVqSyiN9N70bZ3QNPxWpAjKZS08X5PK0teI0b3uMZB9KwCdPkjMxHyymulyQ/bOkm1CCogFpGlG8abwyusIfrrjQOCeX+sUSei6/K/eN2kPLUUcoECgYEA0EKwQPq6mXmXiWqVMk08wz1JPqiNE9zreqaSOx9aOO/DbeR0xkfWjKy4RoCY9zo8FW0Y7jvwuEojm7LNZSrLrur3b2myfzHsdsjuo2t/fVilVrjtzxDEQSeBNOJasVdS+ZhB+/GHWfeZEcMjhXHkW9WRgMwIC/jTanmqLYy4nzkCgYEA37ZZJ+QH8+saafhztaSK/IC8pjJLTkHsMBjhSpVGm9TptpVdxHx/cHJdCDiHPfeP+TIPSOD//hGbHoTZuy2owZ0FJ2QFxyLpagEUch+Sy0f7fQZ+LoeUa1UJ9RsCbCCeKtcZvqZ0fuu9iWoR8LvJ7UI19SEyQ2T7Fymvbq3hFXUCgYEAsHHf7J3BHKjE97rifwSrV2sENF8Pb+W7aGXZ/NdaVGTm+aMWQKu6neL0GV94ufWP1ENjXOxRzYGa255IoM76VM9kJfOyNEuy4QzqCnDYSfWh13DEoqu86sqykIC6gAfRGACk3vVKTLIW8NKYtMXCyP+P0ESNCL+fN1WvFfpkrRkCgYEA3nriP6GvpwxwwGKt8D8rWeJNuprZ+YHl+g9EPoAmMGOV6laxYe7Obm3Nx5cwKJhDPnhiawAYlfu8YKWOQ3AtHB+kOIBonppBt4JLaxOrUS7NFJGYe32qRPPVa0TpK89kfQZePBQeVvrrC/XI0bhwINxv/NB+xDdw3qA+L7wM1OECgYA+3YgYJ6nbvT7s4DylRmyZ4AfOt3TcIjS3D0PDj4S4agZrvlPCKgn1LwBIFutjOe6V9lPSWZJiw6flfImhpWvQOap3zdT9gsU5PhbUmD3TndAk0n10+lmMYpGkU2UM/+zV5HsiZA5X+DPcVbSnIuNT/7pZdbD5uFF9Ibzoh022Fg==',
+  pubKey: 'CAASpgIwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQC1/m95i4waplpSl43/uefQClPfuguPsX6qa6pWE0Df+krs5p+RnhX+8j2aWBI9QZ0pliGryet5qM3in6J1ihb2LzjLUWIsWrMLxtqi/mQDjj1f0HfV11Q/k2v6yVgSgYJ+VoW51hCpAEdEnhthsbyKWMD6xVIvtkqOmK/FaxUMa2boWCVKsycf39U6GVaiuTk32btw6LqGMo8P1cehWDy80SOzv0qk2mD7ZJbKnVfVm8xwU+gzogasOFYs/LmKb4IEV9Otq0MHmWGUN0VyC82htoGBidheRs1Ssh7TDVtWlNTJSG1vJJV7cnIhpGKut2b0pMxhY1Sg0kn/VrfriXINAgMBAAE='
```

Comment:
@whyrusleeping, just catching your attention here to say that we are moving to the way keys are stored in go-ipfs. Would you consider later moving back to the PEM format? it is way more nicer to other languages and more intuitive.